includes/discovery/sensors/count/printer.inc.php

```diff
@@ -73,3 +73,19 @@
         }
     }
 }
+
+if ($device['os'] == 'sharp') {
+    $oids = SnmpQuery::walk('SNMPv2-SMI::enterprises.2385.1.1.19.2.1')->table();
+    $valuesData = $oids['SNMPv2-SMI::enterprises'][2385][1][1][19][2][1][3];
+    $namesData = $oids['SNMPv2-SMI::enterprises'][2385][1][1][19][2][1][4];
+    foreach ($namesData as $index1 => $nData1) {
+        foreach ($nData1 as $index2 => $nData2) {
+            foreach ($nData2 as $index3 => $sensorName) {
```

Comment:
Not possible to do it in YAML ?

Reply:
> Not possible to do it in YAML ?

if you try to google for
.1.3.6.1.4.1.2385.1.1.19.2.1.3.
you will find nothing
sharp does not have MIBs
and without MIBs, there is no chance to implement this in YAML, agree ?

Reply:
Agreed. But I could not know that until I asked you :)